app/models/school.rb

```diff
@@ -74,6 +74,14 @@ def is_further_education?
     type == 'FurtherEducationSchool'
   end
 
+  def available_mobile_networks
+    if hide_mno?
```

Comment:
I wonder if we should have something like this?

```ruby
def hide_networks_not_supporting_fe?
  hide_mno?
end```

`hide_mno` doesn't do the heavy lifting in explaining

Reply:
yea probably a good call. i did consider renaming the entire column. Aliasing seems a good middle ground